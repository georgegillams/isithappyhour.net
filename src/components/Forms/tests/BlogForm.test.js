import React from 'react';
import { render } from '@testing-library/react';

import { BlogForm } from '..';

describe('<BlogForm />', () => {
  it('Should render correctly', () => {
    const { container } = render(
      <BlogForm blog={{ id: 'some-blog' }} onDataChanged={() => null} onSubmit={() => null} />
    );

    expect(container).toMatchSnapshot();
  });

  it('Should render correctly with no blog id', () => {
    const { container } = render(<BlogForm blog={{}} onDataChanged={() => null} onSubmit={() => null} />);

    expect(container).toMatchSnapshot();
  });
});
