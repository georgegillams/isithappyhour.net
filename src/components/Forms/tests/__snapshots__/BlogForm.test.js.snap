// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BlogForm /> Should render correctly 1`] = `
<div>
  <div
    class="sc-crXcEl"
  >
    <label
      class="sc-fEOsli iHSlJz"
      for="title_FORM_ID"
    >
      Title
    </label>
    <div
      class="sc-iBkjds hDuMmX sc-breuTD bOZwsL"
      placeholder="Title"
    >
      <input
        aria-disabled="false"
        aria-invalid="false"
        class="sc-dIouRR cqKYoJ"
        id="title_FORM_ID"
        name="Title"
        value=""
      />
      <svg
        class="sc-gKXOVf ktoyia sc-hHLeRK kBmlVx"
        height="18"
        viewBox="0 0 24 24"
        width="18"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M9.4 20c-.5 0-.9-.2-1.3-.5l-5.8-5.1c-.4-.4-.5-1-.1-1.4l1.3-1.5c.4-.4 1-.5 1.4-.1l4 3.5c.2.1.4.1.6 0l9.2-10.5c.4-.4 1-.5 1.4-.1l1.5 1.3c.4.4.5 1 .1 1.4L10.9 19.3c-.4.5-.9.7-1.5.7z"
        />
      </svg>
    </div>
    <label
      class="sc-fEOsli iHSlJz"
      for="tags_FORM_ID"
    >
      Tags
    </label>
    <div
      class="sc-iBkjds hDuMmX sc-breuTD bOZwsL"
      placeholder="Tags"
    >
      <input
        aria-disabled="false"
        aria-invalid="false"
        class="sc-dIouRR cqKYoJ"
        id="tags_FORM_ID"
        name="Tags"
        value=""
      />
      <svg
        class="sc-gKXOVf ktoyia sc-hHLeRK kBmlVx"
        height="18"
        viewBox="0 0 24 24"
        width="18"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M9.4 20c-.5 0-.9-.2-1.3-.5l-5.8-5.1c-.4-.4-.5-1-.1-1.4l1.3-1.5c.4-.4 1-.5 1.4-.1l4 3.5c.2.1.4.1.6 0l9.2-10.5c.4-.4 1-.5 1.4-.1l1.5 1.3c.4.4.5 1 .1 1.4L10.9 19.3c-.4.5-.9.7-1.5.7z"
        />
      </svg>
    </div>
    <label
      class="sc-fEOsli iHSlJz"
      for="blogImage_FORM_ID"
    >
      Blog image
    </label>
    <div
      class="sc-iBkjds hDuMmX sc-breuTD bOZwsL"
      placeholder="Blog image"
    >
      <input
        aria-disabled="false"
        aria-invalid="false"
        class="sc-dIouRR cqKYoJ"
        id="blogImage_FORM_ID"
        name="Blog image"
        value=""
      />
      <svg
        class="sc-gKXOVf ktoyia sc-hHLeRK kBmlVx"
        height="18"
        viewBox="0 0 24 24"
        width="18"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M9.4 20c-.5 0-.9-.2-1.3-.5l-5.8-5.1c-.4-.4-.5-1-.1-1.4l1.3-1.5c.4-.4 1-.5 1.4-.1l4 3.5c.2.1.4.1.6 0l9.2-10.5c.4-.4 1-.5 1.4-.1l1.5 1.3c.4.4.5 1 .1 1.4L10.9 19.3c-.4.5-.9.7-1.5.7z"
        />
      </svg>
    </div>
    <label
      class="sc-fEOsli iHSlJz"
      for="publishedTimestamp_FORM_ID"
    >
      Published timestamp
    </label>
    <div
      class="sc-iBkjds hDuMmX sc-breuTD bOZwsL"
      placeholder="Published timestamp"
    >
      <input
        aria-disabled="false"
        aria-invalid="false"
        class="sc-dIouRR cqKYoJ"
        id="publishedTimestamp_FORM_ID"
        name="Published timestamp"
        value=""
      />
      <svg
        class="sc-gKXOVf ktoyia sc-hHLeRK kBmlVx"
        height="18"
        viewBox="0 0 24 24"
        width="18"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M9.4 20c-.5 0-.9-.2-1.3-.5l-5.8-5.1c-.4-.4-.5-1-.1-1.4l1.3-1.5c.4-.4 1-.5 1.4-.1l4 3.5c.2.1.4.1.6 0l9.2-10.5c.4-.4 1-.5 1.4-.1l1.5 1.3c.4.4.5 1 .1 1.4L10.9 19.3c-.4.5-.9.7-1.5.7z"
        />
      </svg>
    </div>
    <label
      class="sc-fEOsli iHSlJz"
      for="blogCardDate_FORM_ID"
    >
      Blog card date override
    </label>
    <div
      class="sc-iBkjds hDuMmX sc-breuTD bOZwsL"
      placeholder="Blog card date override"
    >
      <input
        aria-disabled="false"
        aria-invalid="false"
        class="sc-dIouRR cqKYoJ"
        id="blogCardDate_FORM_ID"
        name="Blog card date override"
        value=""
      />
      <svg
        class="sc-gKXOVf ktoyia sc-hHLeRK kBmlVx"
        height="18"
        viewBox="0 0 24 24"
        width="18"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M9.4 20c-.5 0-.9-.2-1.3-.5l-5.8-5.1c-.4-.4-.5-1-.1-1.4l1.3-1.5c.4-.4 1-.5 1.4-.1l4 3.5c.2.1.4.1.6 0l9.2-10.5c.4-.4 1-.5 1.4-.1l1.5 1.3c.4.4.5 1 .1 1.4L10.9 19.3c-.4.5-.9.7-1.5.7z"
        />
      </svg>
    </div>
    <label
      class="sc-ftvSup bMRhqZ sc-fnykZs dYLQUI"
      id="light_FORM_ID"
    >
      <div
        class="sc-papXJ gtFCxX"
      >
        <input
          aria-invalid="false"
          aria-label="Light"
          class="sc-jqUVSM hocLPL"
          name="Light"
          type="checkbox"
        />
        <svg
          class="sc-gKXOVf ktoyia sc-kDDrLX bmdoJK"
          height="18"
          viewBox="0 0 24 24"
          width="18"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M9.4 20c-.5 0-.9-.2-1.3-.5l-5.8-5.1c-.4-.4-.5-1-.1-1.4l1.3-1.5c.4-.4 1-.5 1.4-.1l4 3.5c.2.1.4.1.6 0l9.2-10.5c.4-.4 1-.5 1.4-.1l1.5 1.3c.4.4.5 1 .1 1.4L10.9 19.3c-.4.5-.9.7-1.5.7z"
          />
        </svg>
      </div>
      <div
        aria-hidden="true"
        class="sc-iqcoie bBXilN"
      >
        Light
      </div>
    </label>
    <label
      class="sc-ftvSup bMRhqZ sc-fnykZs dYLQUI"
      id="published_FORM_ID"
    >
      <div
        class="sc-papXJ gtFCxX"
      >
        <input
          aria-invalid="false"
          aria-label="Published"
          class="sc-jqUVSM hocLPL"
          name="Published"
          type="checkbox"
        />
        <svg
          class="sc-gKXOVf ktoyia sc-kDDrLX bmdoJK"
          height="18"
          viewBox="0 0 24 24"
          width="18"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M9.4 20c-.5 0-.9-.2-1.3-.5l-5.8-5.1c-.4-.4-.5-1-.1-1.4l1.3-1.5c.4-.4 1-.5 1.4-.1l4 3.5c.2.1.4.1.6 0l9.2-10.5c.4-.4 1-.5 1.4-.1l1.5 1.3c.4.4.5 1 .1 1.4L10.9 19.3c-.4.5-.9.7-1.5.7z"
          />
        </svg>
      </div>
      <div
        aria-hidden="true"
        class="sc-iqcoie bBXilN"
      >
        Published
      </div>
    </label>
    <label
      class="sc-ftvSup bMRhqZ sc-fnykZs dYLQUI"
      id="showInBlogsList_FORM_ID"
    >
      <div
        class="sc-papXJ gtFCxX"
      >
        <input
          aria-invalid="false"
          aria-label="Show in blogs list"
          class="sc-jqUVSM hocLPL"
          name="Show in blogs list"
          type="checkbox"
        />
        <svg
          class="sc-gKXOVf ktoyia sc-kDDrLX bmdoJK"
          height="18"
          viewBox="0 0 24 24"
          width="18"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M9.4 20c-.5 0-.9-.2-1.3-.5l-5.8-5.1c-.4-.4-.5-1-.1-1.4l1.3-1.5c.4-.4 1-.5 1.4-.1l4 3.5c.2.1.4.1.6 0l9.2-10.5c.4-.4 1-.5 1.4-.1l1.5 1.3c.4.4.5 1 .1 1.4L10.9 19.3c-.4.5-.9.7-1.5.7z"
          />
        </svg>
      </div>
      <div
        aria-hidden="true"
        class="sc-iqcoie bBXilN"
      >
        Show in blogs list
      </div>
    </label>
    <label
      class="sc-fEOsli iHSlJz"
      for="blogImageBorderColor_FORM_ID"
    >
      Blog image border color
    </label>
    <div
      class="sc-iBkjds hDuMmX sc-breuTD bOZwsL"
      placeholder="Blog image border color"
    >
      <input
        aria-disabled="false"
        aria-invalid="false"
        class="sc-dIouRR cqKYoJ"
        id="blogImageBorderColor_FORM_ID"
        name="Blog image border color"
        value=""
      />
      <svg
        class="sc-gKXOVf ktoyia sc-hHLeRK kBmlVx"
        height="18"
        viewBox="0 0 24 24"
        width="18"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M9.4 20c-.5 0-.9-.2-1.3-.5l-5.8-5.1c-.4-.4-.5-1-.1-1.4l1.3-1.5c.4-.4 1-.5 1.4-.1l4 3.5c.2.1.4.1.6 0l9.2-10.5c.4-.4 1-.5 1.4-.1l1.5 1.3c.4.4.5 1 .1 1.4L10.9 19.3c-.4.5-.9.7-1.5.7z"
        />
      </svg>
    </div>
    <label
      class="sc-fEOsli iHSlJz"
      for="content_FORM_ID"
    >
      Content
    </label>
    <div
      class="sc-iBkjds hDuMmX sc-ksZaOG gYTEOI"
      placeholder="Content"
    >
      <textarea
        aria-disabled="false"
        aria-invalid="false"
        class="sc-dmRaPn gAvRkq"
        id="content_FORM_ID"
        name="Content"
      />
      <svg
        class="sc-gKXOVf ktoyia sc-hHLeRK kBmlVx"
        height="18"
        viewBox="0 0 24 24"
        width="18"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M9.4 20c-.5 0-.9-.2-1.3-.5l-5.8-5.1c-.4-.4-.5-1-.1-1.4l1.3-1.5c.4-.4 1-.5 1.4-.1l4 3.5c.2.1.4.1.6 0l9.2-10.5c.4-.4 1-.5 1.4-.1l1.5 1.3c.4.4.5 1 .1 1.4L10.9 19.3c-.4.5-.9.7-1.5.7z"
        />
      </svg>
    </div>
    <button
      class="sc-dkzDqf bOfPfm sc-evZas hAEDCd"
      type="button"
    >
      <span
        class="sc-eCYdqJ ghZAyu"
      />
    </button>
  </div>
</div>
`;

exports[`<BlogForm /> Should render correctly with no blog id 1`] = `
<div>
  <div
    class="sc-crXcEl"
  >
    <label
      class="sc-fEOsli iHSlJz"
      for="requestedId_FORM_ID"
    >
      Requested ID
    </label>
    <div
      class="sc-iBkjds hDuMmX sc-breuTD bOZwsL"
      placeholder="Requested ID"
    >
      <input
        aria-disabled="false"
        aria-invalid="false"
        class="sc-dIouRR cqKYoJ"
        id="requestedId_FORM_ID"
        name="Requested ID"
        value=""
      />
      <svg
        class="sc-gKXOVf ktoyia sc-hHLeRK kBmlVx"
        height="18"
        viewBox="0 0 24 24"
        width="18"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M9.4 20c-.5 0-.9-.2-1.3-.5l-5.8-5.1c-.4-.4-.5-1-.1-1.4l1.3-1.5c.4-.4 1-.5 1.4-.1l4 3.5c.2.1.4.1.6 0l9.2-10.5c.4-.4 1-.5 1.4-.1l1.5 1.3c.4.4.5 1 .1 1.4L10.9 19.3c-.4.5-.9.7-1.5.7z"
        />
      </svg>
    </div>
    <label
      class="sc-fEOsli iHSlJz"
      for="title_FORM_ID"
    >
      Title
    </label>
    <div
      class="sc-iBkjds hDuMmX sc-breuTD bOZwsL"
      placeholder="Title"
    >
      <input
        aria-disabled="false"
        aria-invalid="false"
        class="sc-dIouRR cqKYoJ"
        id="title_FORM_ID"
        name="Title"
        value=""
      />
      <svg
        class="sc-gKXOVf ktoyia sc-hHLeRK kBmlVx"
        height="18"
        viewBox="0 0 24 24"
        width="18"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M9.4 20c-.5 0-.9-.2-1.3-.5l-5.8-5.1c-.4-.4-.5-1-.1-1.4l1.3-1.5c.4-.4 1-.5 1.4-.1l4 3.5c.2.1.4.1.6 0l9.2-10.5c.4-.4 1-.5 1.4-.1l1.5 1.3c.4.4.5 1 .1 1.4L10.9 19.3c-.4.5-.9.7-1.5.7z"
        />
      </svg>
    </div>
    <label
      class="sc-fEOsli iHSlJz"
      for="tags_FORM_ID"
    >
      Tags
    </label>
    <div
      class="sc-iBkjds hDuMmX sc-breuTD bOZwsL"
      placeholder="Tags"
    >
      <input
        aria-disabled="false"
        aria-invalid="false"
        class="sc-dIouRR cqKYoJ"
        id="tags_FORM_ID"
        name="Tags"
        value=""
      />
      <svg
        class="sc-gKXOVf ktoyia sc-hHLeRK kBmlVx"
        height="18"
        viewBox="0 0 24 24"
        width="18"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M9.4 20c-.5 0-.9-.2-1.3-.5l-5.8-5.1c-.4-.4-.5-1-.1-1.4l1.3-1.5c.4-.4 1-.5 1.4-.1l4 3.5c.2.1.4.1.6 0l9.2-10.5c.4-.4 1-.5 1.4-.1l1.5 1.3c.4.4.5 1 .1 1.4L10.9 19.3c-.4.5-.9.7-1.5.7z"
        />
      </svg>
    </div>
    <label
      class="sc-fEOsli iHSlJz"
      for="blogImage_FORM_ID"
    >
      Blog image
    </label>
    <div
      class="sc-iBkjds hDuMmX sc-breuTD bOZwsL"
      placeholder="Blog image"
    >
      <input
        aria-disabled="false"
        aria-invalid="false"
        class="sc-dIouRR cqKYoJ"
        id="blogImage_FORM_ID"
        name="Blog image"
        value=""
      />
      <svg
        class="sc-gKXOVf ktoyia sc-hHLeRK kBmlVx"
        height="18"
        viewBox="0 0 24 24"
        width="18"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M9.4 20c-.5 0-.9-.2-1.3-.5l-5.8-5.1c-.4-.4-.5-1-.1-1.4l1.3-1.5c.4-.4 1-.5 1.4-.1l4 3.5c.2.1.4.1.6 0l9.2-10.5c.4-.4 1-.5 1.4-.1l1.5 1.3c.4.4.5 1 .1 1.4L10.9 19.3c-.4.5-.9.7-1.5.7z"
        />
      </svg>
    </div>
    <label
      class="sc-fEOsli iHSlJz"
      for="publishedTimestamp_FORM_ID"
    >
      Published timestamp
    </label>
    <div
      class="sc-iBkjds hDuMmX sc-breuTD bOZwsL"
      placeholder="Published timestamp"
    >
      <input
        aria-disabled="false"
        aria-invalid="false"
        class="sc-dIouRR cqKYoJ"
        id="publishedTimestamp_FORM_ID"
        name="Published timestamp"
        value=""
      />
      <svg
        class="sc-gKXOVf ktoyia sc-hHLeRK kBmlVx"
        height="18"
        viewBox="0 0 24 24"
        width="18"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M9.4 20c-.5 0-.9-.2-1.3-.5l-5.8-5.1c-.4-.4-.5-1-.1-1.4l1.3-1.5c.4-.4 1-.5 1.4-.1l4 3.5c.2.1.4.1.6 0l9.2-10.5c.4-.4 1-.5 1.4-.1l1.5 1.3c.4.4.5 1 .1 1.4L10.9 19.3c-.4.5-.9.7-1.5.7z"
        />
      </svg>
    </div>
    <label
      class="sc-fEOsli iHSlJz"
      for="blogCardDate_FORM_ID"
    >
      Blog card date override
    </label>
    <div
      class="sc-iBkjds hDuMmX sc-breuTD bOZwsL"
      placeholder="Blog card date override"
    >
      <input
        aria-disabled="false"
        aria-invalid="false"
        class="sc-dIouRR cqKYoJ"
        id="blogCardDate_FORM_ID"
        name="Blog card date override"
        value=""
      />
      <svg
        class="sc-gKXOVf ktoyia sc-hHLeRK kBmlVx"
        height="18"
        viewBox="0 0 24 24"
        width="18"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M9.4 20c-.5 0-.9-.2-1.3-.5l-5.8-5.1c-.4-.4-.5-1-.1-1.4l1.3-1.5c.4-.4 1-.5 1.4-.1l4 3.5c.2.1.4.1.6 0l9.2-10.5c.4-.4 1-.5 1.4-.1l1.5 1.3c.4.4.5 1 .1 1.4L10.9 19.3c-.4.5-.9.7-1.5.7z"
        />
      </svg>
    </div>
    <label
      class="sc-ftvSup bMRhqZ sc-fnykZs dYLQUI"
      id="light_FORM_ID"
    >
      <div
        class="sc-papXJ gtFCxX"
      >
        <input
          aria-invalid="false"
          aria-label="Light"
          class="sc-jqUVSM hocLPL"
          name="Light"
          type="checkbox"
        />
        <svg
          class="sc-gKXOVf ktoyia sc-kDDrLX bmdoJK"
          height="18"
          viewBox="0 0 24 24"
          width="18"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M9.4 20c-.5 0-.9-.2-1.3-.5l-5.8-5.1c-.4-.4-.5-1-.1-1.4l1.3-1.5c.4-.4 1-.5 1.4-.1l4 3.5c.2.1.4.1.6 0l9.2-10.5c.4-.4 1-.5 1.4-.1l1.5 1.3c.4.4.5 1 .1 1.4L10.9 19.3c-.4.5-.9.7-1.5.7z"
          />
        </svg>
      </div>
      <div
        aria-hidden="true"
        class="sc-iqcoie bBXilN"
      >
        Light
      </div>
    </label>
    <label
      class="sc-ftvSup bMRhqZ sc-fnykZs dYLQUI"
      id="published_FORM_ID"
    >
      <div
        class="sc-papXJ gtFCxX"
      >
        <input
          aria-invalid="false"
          aria-label="Published"
          class="sc-jqUVSM hocLPL"
          name="Published"
          type="checkbox"
        />
        <svg
          class="sc-gKXOVf ktoyia sc-kDDrLX bmdoJK"
          height="18"
          viewBox="0 0 24 24"
          width="18"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M9.4 20c-.5 0-.9-.2-1.3-.5l-5.8-5.1c-.4-.4-.5-1-.1-1.4l1.3-1.5c.4-.4 1-.5 1.4-.1l4 3.5c.2.1.4.1.6 0l9.2-10.5c.4-.4 1-.5 1.4-.1l1.5 1.3c.4.4.5 1 .1 1.4L10.9 19.3c-.4.5-.9.7-1.5.7z"
          />
        </svg>
      </div>
      <div
        aria-hidden="true"
        class="sc-iqcoie bBXilN"
      >
        Published
      </div>
    </label>
    <label
      class="sc-ftvSup bMRhqZ sc-fnykZs dYLQUI"
      id="showInBlogsList_FORM_ID"
    >
      <div
        class="sc-papXJ gtFCxX"
      >
        <input
          aria-invalid="false"
          aria-label="Show in blogs list"
          class="sc-jqUVSM hocLPL"
          name="Show in blogs list"
          type="checkbox"
        />
        <svg
          class="sc-gKXOVf ktoyia sc-kDDrLX bmdoJK"
          height="18"
          viewBox="0 0 24 24"
          width="18"
          xmlns="http://www.w3.org/2000/svg"
        >
          <path
            d="M9.4 20c-.5 0-.9-.2-1.3-.5l-5.8-5.1c-.4-.4-.5-1-.1-1.4l1.3-1.5c.4-.4 1-.5 1.4-.1l4 3.5c.2.1.4.1.6 0l9.2-10.5c.4-.4 1-.5 1.4-.1l1.5 1.3c.4.4.5 1 .1 1.4L10.9 19.3c-.4.5-.9.7-1.5.7z"
          />
        </svg>
      </div>
      <div
        aria-hidden="true"
        class="sc-iqcoie bBXilN"
      >
        Show in blogs list
      </div>
    </label>
    <label
      class="sc-fEOsli iHSlJz"
      for="blogImageBorderColor_FORM_ID"
    >
      Blog image border color
    </label>
    <div
      class="sc-iBkjds hDuMmX sc-breuTD bOZwsL"
      placeholder="Blog image border color"
    >
      <input
        aria-disabled="false"
        aria-invalid="false"
        class="sc-dIouRR cqKYoJ"
        id="blogImageBorderColor_FORM_ID"
        name="Blog image border color"
        value=""
      />
      <svg
        class="sc-gKXOVf ktoyia sc-hHLeRK kBmlVx"
        height="18"
        viewBox="0 0 24 24"
        width="18"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M9.4 20c-.5 0-.9-.2-1.3-.5l-5.8-5.1c-.4-.4-.5-1-.1-1.4l1.3-1.5c.4-.4 1-.5 1.4-.1l4 3.5c.2.1.4.1.6 0l9.2-10.5c.4-.4 1-.5 1.4-.1l1.5 1.3c.4.4.5 1 .1 1.4L10.9 19.3c-.4.5-.9.7-1.5.7z"
        />
      </svg>
    </div>
    <label
      class="sc-fEOsli iHSlJz"
      for="content_FORM_ID"
    >
      Content
    </label>
    <div
      class="sc-iBkjds hDuMmX sc-ksZaOG gYTEOI"
      placeholder="Content"
    >
      <textarea
        aria-disabled="false"
        aria-invalid="false"
        class="sc-dmRaPn gAvRkq"
        id="content_FORM_ID"
        name="Content"
      />
      <svg
        class="sc-gKXOVf ktoyia sc-hHLeRK kBmlVx"
        height="18"
        viewBox="0 0 24 24"
        width="18"
        xmlns="http://www.w3.org/2000/svg"
      >
        <path
          d="M9.4 20c-.5 0-.9-.2-1.3-.5l-5.8-5.1c-.4-.4-.5-1-.1-1.4l1.3-1.5c.4-.4 1-.5 1.4-.1l4 3.5c.2.1.4.1.6 0l9.2-10.5c.4-.4 1-.5 1.4-.1l1.5 1.3c.4.4.5 1 .1 1.4L10.9 19.3c-.4.5-.9.7-1.5.7z"
        />
      </svg>
    </div>
    <button
      class="sc-dkzDqf bOfPfm sc-evZas hAEDCd"
      type="button"
    >
      <span
        class="sc-eCYdqJ ghZAyu"
      />
    </button>
  </div>
</div>
`;
